### PR TITLE
started recipe for braggedgemodeling

### PR DESCRIPTION
**Pull request by yxqd**
https://github.com/ornlneutronimaging/braggedgemodeling/

### CODE REVIEW

**Comment by yxqd** [comment]
@ocefpaf It seems there is a problem with conda-smithy/lint_recipe.py right now at
https://github.com/conda-forge/conda-smithy/blob/5ba8b1fd3d2f288f4265c7538ec95b4bcf881a48/conda_smithy/lint_recipe.py#L135

The code there expect `extra_section.get('recipe-maintainers')` to be a list, but it is right now an instance of `ruamel.yaml.comments.CommentedSeq`. Probably `ruamel.yaml` has a new version and has changed its types?

**Comment by ocefpaf** [comment]
> Probably ruamel.yaml has a new version and has changed its types?

Probably. It is safe to ignore that BTW as it is ultimately a bug in our linter.

File: recipes/braggedgemodeling/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{% set name = "braggedgemodeling" %}
+{% set version = "0.1.1a2" %}
+{% set git_rev = "v0.1.1a2" %}
```
Comment (ocefpaf) [inline recipes/braggedgemodeling/meta.yaml]: Our bot won't be able to update your package this way. You can set `version` and then just leave the `v` in the URL below removing he need for this extra variable.

File: recipes/braggedgemodeling/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    
+test:
+  imports:
+    - bem
+
+about:
+  home: https://github.com/ornlneutronimaging/braggedgemodeling
+  license: MIT
+  license_family: MIT
+  # license_file: LICENSE
```
Comment (ocefpaf) [inline recipes/braggedgemodeling/meta.yaml]: Please open an issue upstream and, if possible, package the license manually:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

**Comment by yxqd** [comment]
@ocefpaf could you please check again? I added a LICENSE and removed the extra variable

File: recipes/braggedgemodeling/meta.yaml [hunk 3]
```diff
@@ -49,3 +48,7 @@ about:
 
 extra:
   recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
+    - LisaSimpson
+    - LandoCalrissian
```
Comment (ocefpaf) [inline recipes/braggedgemodeling/meta.yaml]: These must be populated with real GitHub handles of people who will maintain the recipe.

Comment (yxqd) [reply recipes/braggedgemodeling/meta.yaml]: already fixed. thanks

File: recipes/braggedgemodeling/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  dev_url: https://github.com/ornlneutronimaging/braggedgemodeling
+  description: |
+    braggedgemodeling is a Python package providing data analysis tools for
+    neutron Bragg Edge spectra.
+    Given material lattice structure, it can compute Bragg Edge transmission
+    spectrum as a function of neutron wavelength.
+
+extra:
+  recipe-maintainers:
+    - yxqd
```
Comment (yxqd) [inline recipes/braggedgemodeling/meta.yaml]: @ocefpaf could you please check this? Thanks

**Comment by yxqd** [comment]
Thanks @ocefpaf !

File: recipes/braggedgemodeling/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,50 @@
+{% set version = "0.1.1a3" %}
+
+package:
+  name: {{ name|lower }}
```
Comment (jakirkham) [inline recipes/braggedgemodeling/meta.yaml]: The `name` variable appears to be undefined. Guessing this was an oversight. Could someone please submit a PR to correct this?

Comment (yxqd) [reply recipes/braggedgemodeling/meta.yaml]: Please see https://github.com/conda-forge/staged-recipes/pull/6533

Comment (jakirkham) [reply recipes/braggedgemodeling/meta.yaml]: Thanks :)

**Pull request merged**